tests/testutil/service.go

```diff
@@ -37,14 +40,27 @@ func (ts *TestService) Create(t *testing.T, source string) {
 	if source == "" {
 		source = "nginx"
 	}
-	_, err := RioCmd([]string{"--namespace", testingNamespace, "run", "-p", "80/http", "-n", ts.AppName, source})
+	args = append(
+		[]string{"--namespace", testingNamespace, "run"},
+		append(args, "-n", ts.AppName, source)...)
+	_, err := RioCmd(args)
 	if err != nil {
 		ts.T.Fatalf("Failed to create service %s: %v", ts.Name, err.Error())
 	}
+	if source[0:4] == "http" && strings.Contains(source, "github") {
```

Comment:
Is there a reason we need to check for both tekton and ready service ? Is it just to get more info. Can you double check also that you can't pass in a non scheme'd url string, like just `github.com/a/b`, as that would need to be taken into account here.

Reply:
non scheme'd url string assumes a docker image (tested and confirmed: `v0: test(ImagePullBackOff/Back-off pulling image "github.com/rancher/rio-demo")`

I'm checking tekton so as not to have to generically increase ready service timeout. When a build happens (passing in GH repo), the time until the service is ready is much much larger, especially with the tests running in parallel, so I was noticing flakiness.